qa/workunits/ceph-helpers.sh

```diff
@@ -146,6 +146,27 @@ function test_teardown() {
 #######################################################################
 
 ##
+# Kills a single daemon. This is an helper function fo kill_daemons
+#
+function kill_daemon() {
+    local pid=$(cat $1)
+    local send_signal=$2
+    local delays=${4:-0.1 0.2 1 1 1 2 3 5 5 5 10 10 20 60 60 60 120}
+    local exit_code=1
+    for try in $delays ; do
+         if kill -$send_signal $pid 2> /dev/null ; then
+            exit_code=1
+         else
+	    exit_code=0
+            break
+         fi
+         send_signal=0
+         sleep $try
+    done;
+    return $exit_code
+}
+
+##
```

Comment:
kill_daemons doesn't do much except calling kill_daemon several time.
The actual test_kill_daemons test the feature pretty well. Having that test passing implies that kill_daemon is ok. Seems overkill to me to add such test.